fv3gfs-physics/tests/savepoint/translate/translate_driver.py

```diff
@@ -0,0 +1,48 @@
+from mpi4py import MPI
+
+from fv3core._config import DynamicalCoreConfig
+
+# TODO physics should not depend on fv3core
+# but also, driver tests should not be in physics
+from fv3core.testing.translate_fvdynamics import TranslateFVDynamics
+from fv3core.testing.validation import enable_selective_validation
```

Comment:
I'll message on slack about this, very confused why enable_selective_validation changes the result here. I'm worried this might be revealing a bug somewhere.